back/spec/acceptance/event_files_spec.rb

```diff
@@ -3,35 +3,48 @@
 require 'rails_helper'
 require 'rspec_api_documentation/dsl'
 
-resource 'EventFile' do
-  explanation 'File attachments.'
+resource 'File attachment as legacy EventFile' do
```

Comment:
Do we need `legacy` here? Could we not just say `File attachments for Events` or is there further work and the whole files controller is considered legacy and will be removed at some point?